### PR TITLE
Created landing page and avatar component

### PR DESCRIPTION
**Pull request by nareshpoonia**
Created Landing Page
Made Avatar Component 
Buttons in Landing Page hasn't been styled yet
Once I'll make the button component, then use the same for styling in landing page

### CODE REVIEW

File: styles/docs.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+code {
+  color: blue;
+}
+.large-heading {
+  font-size: 5rem;
+}
+.mgl-2 {
+  margin-left: 2rem;
+}
+.mglr-2 {
```
Comment (FarhanMobashir) [inline styles/docs.css]: Instead of using left right as naming convention use margin horizontal and margin vertical.

**Review by FarhanMobashir — APPROVED** [review]
All good overall. Just go through the comments and make the necessary changes

File: styles/docs.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  margin: 2rem;
+}
+.mg-t-3 {
+  margin-top: 3rem;
+}
+/*  */
+.aside-menu {
+  display: none;
+  background-color: black;
+  /* display: flex; */
```
Comment (lannisterrr) [inline styles/docs.css]: remove comments.

File: styles/index.css [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+body {
+  background-color: var(--background-color);
+  font-family: Arial, Helvetica, sans-serif;
+  color: var(--primary-color);
+}
+
+/* Utility Classses */
+
+.mg-0-2 {
```
Comment (lannisterrr) [inline styles/index.css]: You should design system and have value in terms of 8 or 4 , also you can name your classes margin-vertical  and margin horizontol.

File: styles/docs.css (hunk 1 above)
Comment (nareshpoonia) [reply styles/docs.css]: Changed margin utility classes as per tailwind

File: styles/docs.css (hunk 2 above)
Comment (nareshpoonia) [reply styles/docs.css]: Removed comments

File: styles/index.css (hunk 3 above)
Comment (nareshpoonia) [reply styles/index.css]: Changed margin utility classes as per tailwind

**Pull request merged**